lib/private/Preview/Bitmap.php

```diff
@@ -37,6 +37,20 @@
  * @package OC\Preview
  */
 abstract class Bitmap extends ProviderV2 {
+	/**
+	 * List of MIME types that this preview provider is allowed to process.
+	 *
+	 * These should correspond to the MIME types *identified* by Imagemagick
+	 * for files to be processed by this provider. These do / will not
+	 * necessarily need to match the MIME types stored in the database
+	 * (which are identified by IMimeTypeDetector).
+	 *
+	 * @return string Regular expression
+	 */
+	protected function getAllowedMimeTypes(): string {
+		return '/invalid\/invalid/';
+	}
```

Comment:
```suggestion
	abstract protected function getAllowedMimeTypes(): string;
```
This is the cleanest actually

Reply:
Done. My original intention was to not fail with a TypeError if e.g. a third party app uses this but now that I think that's a better failure mode.